x/gov/migrations/v046/store.go

```diff
@@ -5,7 +5,9 @@ import (
 	"github.com/cosmos/cosmos-sdk/store/prefix"
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/gov/exported"
 	v042 "github.com/cosmos/cosmos-sdk/x/gov/migrations/v042"
+	v1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
 	"github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
 )
 
```

Comment:
I cannot comment higher, but we have been renaming the migrations package as well (`v1`, `v2`, `vX` instead of version name). We should do it here too for consistency. See https://github.com/cosmos/cosmos-sdk/pull/12579#issue-1304814850